src/adapters/sns/publisher.ts

```diff
@@ -0,0 +1,83 @@
+import { Events } from '@dcl/schemas/dist/platform/events'
+import { AppComponents, SnsPublisherComponent, SnsPublishError } from '../../types'
+import { PublishCommand, SNSClient } from '@aws-sdk/client-sns'
+import { DeployableEntity } from '@dcl/snapshots-fetcher/dist/types'
+import { SnsOptions, SnsType } from './types'
+import { buildDeploymentMessage } from '../../logic/build-deployment-message'
+
+export async function createSnsPublisherComponent(
```

Comment:
Do we need to export it?
```suggestion
async function createSnsPublisherComponent(
```